registration/templates/privacy_policy.html

```diff
@@ -1,13 +1,12 @@
 {% extends 'base.html' %}
 
 {% load i18n %}
+{% load include_file %}
 
 {% block content %}
 <h1> {% translate 'Privacy Policy' %} </h2>
     <p>
-        {% blocktranslate %}
-        This is the privacy policy.
-        {% endblocktranslate %}
+        {% include_file 'files/privacy_policy.txt'%}
     </p>
```

Comment:
```suggestion
        {% autoescape off %}{% include_file 'files/privacy_policy.html'%}{% endautoescape %}
```

Reply:
Das gibt ihnen die Möglichkeit, den Text zu formatieren. Die Datei in files sollte dann natürlich noch umbenannt werden.